va/win32/va_win32.c

```diff
@@ -59,7 +59,7 @@ static void LoadDriverNameFromRegistry(VADisplayContextWin32* pWin32Ctx)
 #ifndef _WIN64
     BOOL isWowProcess = false;
     if (IsWow64Process(GetCurrentProcess(), &isWowProcess) && isWowProcess)
-        wcscpy(RegistryInfo.ValueName, "VAAPIDriverNameWow");
+        wcscpy(RegistryInfo.ValueName, L"VAAPIDriverNameWow");
```

Comment:
Oof, yeah I didn't build x86, that's my bad.

Reply:
Heh - we've all been there. That's why the CI use werror=true